src/ui/components/shared/Login/Login.tsx

```diff
@@ -177,6 +177,10 @@ export default function Login({ returnToPath = "" }: { returnToPath?: string })
   const { loginWithRedirect } = useAuth0();
   const [sso, setSSO] = useState(false);
 
+  if (returnToPath.startsWith("/login")) {
```

Comment:
Would this be better handled in `pages/login.tsx`?

Reply:
I wanted to make put it as late in the chain of components as possible so I was confident that we wouldn't be missing a path, since `<Login />` is rendered from quite a few places.

Reply:
I think the only case of the loop would be if it were rendered from the login page.